### ISSUE TITLE
HTML feedback

### ISSUE THREAD

**Issue by jema28** [lotus-1/MediaWikiGiphy#7]
Really clean work 😍  Here are a few comments: 
1. The h1 tag should be before any h2 or h3s, to improve accessibility. Have a look at the lines below:
https://github.com/lotus-1/MediaWikiGiphy/blob/69e85f9a34bb044f1da6ab80895ee82c711c50dc/index.html#L15-L17
2. It's great practice to include a meta tag with [viewport for responsive design ](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag):
`<meta name="viewport" content="width=device-width, initial-scale=1">`

Awesome job and loving the perfect indentation 🎉 😻

**Comment by amanhalabyar** [comment]
Thank you so much for your feedback i appreciate it a lot!
